_posts/2017-06-28-Openshift-an-introduction.md

```diff
@@ -0,0 +1,241 @@
+---
+layout: post
+authors: [hans_michiels]
+title: 'Openshift: An introduction'
+image: /img/openshift.png
+tags: [Openshift, PaaS, Cloud, Container, Devops, Kubernetes]
+category: Paas
+comments: true
+---
+<div class="col-md-2" style="width:32%">
+{% include image.html img="/img/openshift-an introduction/Ease-of-PaaS.png" alt="easeofpaas" title="Paas" style="max-width: 80px;" %}</div>
+
+
+# Why my first blog about Openshift
+When I started as a developer, the cloud ecosystem started expanding and became the next big thing.
+So obvious I wanted to see what all the fuzz was about and started taking a deeper look at it.
+Soon I got introduced with so many new technologies I was not familiar with. People talked about microservices, containers, pods, kubernetes,load balancing, docker,Paas,...
+
+To be honest, for me it was really overwhelming.
+I thought myself I never have the time to become a guru of all these technologies to start with cloud native development.
+So I just sat in a corner crying about why I became a dev in a time where things never looked more complicated and changed faster then ever before.
+But actually it's not at all complicated, to be honest deploying your containers in the cloud and managing things are easier then ever before with PaaS and Openshift.
+
+A couple of months ago, I got introduced to Openshift and got really excited about it!
+Recently Ordina gave me the chance to visit the RedHats partner conference and my excitement for Openshift reached new heights.
+My body not able to contain all that energy for Openshift, I had to ventilate it into a blogpost or otherwise I would spontaneous combust.
+
+I do have to mention when you want to work with openshift, you still need to have a basic understanding about containers, paas and kubernetes if you want to understand some of it's magic.
+If you have no idea what docker containers are or what kubernetes is, don't panic! There are some great blogposts on the jworks blog explaining more about them.
+
+# What is Openshift
+
+Openshift is a __PaaS__. For those who don't know what a PAAS is, stop reading now, take a timecab to the year 2017 and check it out cause PaaS is awesome.
```

Comment:
Openshift is a __PaaS__. For those who don't know what a PaaS is, stop reading now, take a timecab to the year 2011 and check it out cause PaaS is awesome.

Gartner calls OpenShift a **Cloud Enabled Application Platform (CEAP)**.